test/controllers/FileChecksResultsControllerSpec.scala

```diff
@@ -436,8 +444,14 @@ class FileChecksResultsControllerSpec extends FrontEndTestHelper {
     setConsignmentTypeResponse(wiremockServer, consignmentType)
   }
 
-  private def instantiateController(securityComponent: SecurityComponents, keycloakConfiguration: KeycloakConfiguration, blockDraftMetadataUpload: Boolean = false) = {
+  private def instantiateController(
+      securityComponent: SecurityComponents,
+      keycloakConfiguration: KeycloakConfiguration,
+      blockDraftMetadataUpload: Boolean = false,
+      blockAutomateJudgmentTransfers: Boolean = true
```

Comment:
Should set the default for the `blockAutomateJudgmentTransfers` to `false` as this will be the state of the code once the feature is fully released and the FAB is removed.